odoo/addons/test_main_flows/static/tests/tours/main_flow.js

```diff
@@ -171,7 +171,7 @@ tour.stepUtils.autoExpandMoreButtons('.o_form_readonly'),
     position: "right",
 }, {
     mobile: true,
-    trigger: ".modal-dialog .o_field_widget[name=name] input",
+    trigger: "input[name=name][placeholder='e.g. Brandom Freeman']:not(.o_invisible_modifier)",
```

Comment:
The placeholder is definitely not an acceptable attribute to hang a selector on.